core/app/modules/kit-library/assets/js/components/item-header.js

```diff
@@ -62,8 +64,8 @@ function useKitCallToActionButton( model, { apply, isApplyLoading, onConnect } )
 			variant: 'contained',
 			color: isApplyLoading ? 'disabled' : 'primary',
 			size: 'sm',
-			onClick: isApplyLoading ? null : apply,
 			includeHeaderBtnClass: false,
+			onClick: eventTracking(),
```

Comment:
You're immediately calling `eventTracking()`

Should be:
`onClick: () => { if (eventTracking) eventTracking() }`

Reply:
Cant make it work, this is one of the four events that doesn't work,

Reply:
Working.